canonical-web-and-design/vanilla-framework
# Built new status labels

## Done
- Built a new set of `status` buttons
  - `p-button--validated`
  - `p-button--in-progress`
  - `p-button--new`
  - `p-button--updated`
  - `p-button--depreacated`
- Added   `@include vf-p-status;` to _vanilla.scss file
- Created seperate example pages for each button rather them all being in the same file

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check new buttons look like the desired design and are unclickable
  - http://0.0.0.0:8101/examples/patterns/status/validated/
  - http://0.0.0.0:8101/examples/patterns/status/in-progress/
  - http://0.0.0.0:8101/examples/patterns/status/new/
  - http://0.0.0.0:8101/examples/patterns/status/updated/
  - http://0.0.0.0:8101/examples/patterns/status/deprecated/
- Check documentation page
  - https://vanilla-framework-canonical-web-and-design-pr-2416.run.demo.haus/patterns/labels/

## Details
Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/2412

## Screenshots
<img width="102" alt="Screenshot 2019-07-23 at 12 06 20" src="https://user-images.githubusercontent.com/17748020/61707428-637b9100-ad42-11e9-881f-707763982812.png">
<img width="114" alt="Screenshot 2019-07-23 at 12 06 09" src="https://user-images.githubusercontent.com/17748020/61707427-637b9100-ad42-11e9-911e-0682e6822669.png">
<img width="68" alt="Screenshot 2019-07-23 at 12 06 33" src="https://user-images.githubusercontent.com/17748020/61707429-637b9100-ad42-11e9-9f9e-c243b6d34e59.png">
<img width="97" alt="Screenshot 2019-07-23 at 12 06 56" src="https://user-images.githubusercontent.com/17748020/61707431-64142780-ad42-11e9-89b0-94ac0bac720f.png">
<img width="118" alt="Screenshot 2019-07-23 at 12 05 47" src="https://user-images.githubusercontent.com/17748020/61707425-637b9100-ad42-11e9-81ff-52b44131ffa6.png">
